workflow-job

```diff
@@ -399,6 +416,43 @@ flows["${sha1}"] = {
             sh "cd dockerfiles/; docker run ${verbose} ${tmpfsOptions} ${allVolumes} ${testNameDefinitions} ${partialBuildEnv} ${images.jamesCompile} ${mergeBranch}"
             testingJava8.success();
 
+            //Deploying Guice Server with Cassandra + RabbitMQ + Swift
+            sh "cd dockerfiles/run/guice/cassandra-rabbitmq; docker cp ${containers.keystore}:${keystorePath} destination/conf"
+            sh "cd dockerfiles/run/guice/cassandra-rabbitmq; docker cp ${containers.jamesTest}:${jarPathCassandraRabbitMQ} destination"
+            sh "cd dockerfiles/run/guice/cassandra-rabbitmq; docker cp ${containers.jamesTest}:${libPathCassandraRabbitMQ} destination"
+            sh "cd dockerfiles/run/guice/cassandra-rabbitmq; docker cp ${containers.jamesTest}:${cliJarPathCassandraRabbitMQ} destination"
+            sh "cd dockerfiles/run/guice/cassandra-rabbitmq; docker cp ${containers.jamesTest}:${clilibPathCassandraRabbitMQ} destination"
+
+            sh "docker run --detach=true --name=${containers.cassandra} ${images.cassandra}"
```

Comment:
This command is later on launched for `cassandra-guice` validation.

Two things:
 - Launching two times a container with the same name will fail
 - We *could* be launching cassandra + elasticsearch only one time to save build time